sdk/core/src/request_options/content_language.rs

```diff
@@ -1,23 +1,48 @@
 use crate::headers::{self, Header};
 
-#[derive(Debug, Clone, Copy)]
-pub struct ContentLanguage<'a>(&'a str);
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ContentLanguage(std::borrow::Cow<'static, str>);
 
-impl<'a, S> From<S> for ContentLanguage<'a>
-where
-    S: Into<&'a str>,
-{
-    fn from(s: S) -> Self {
-        Self(s.into())
+impl ContentLanguage {
+    pub fn as_str(&self) -> &str {
+        self.0.as_ref()
+    }
+
+    const fn from_static(s: &'static str) -> Self {
+        Self(std::borrow::Cow::Borrowed(s))
+    }
+}
+
+impl From<&'static str> for ContentLanguage {
```

Comment:
I'm new to `const fn`. Would it be better for remove this and require the use of `ContentLanguage::from_static` which is a `const fn`? In general, are `const fn` able to provide optimations?